index.css

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Poppins&display=swap');
+@import url(style.css);
+
+* {
+    box-sizing: border-box;
+    padding: 0rem;
+    margin: 0rem;
+}
+
+body {
+    font-family: 'Montserrat',
+        sans-serif;
+    font-family: 'Poppins',
```

Comment:
The second `font-family` will overwrite the first one, you can remove either one